.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: ruby
 sudo: required
+dist: xenial
```

Comment:
Need upgrading ubuntu version to use new version docker.
In trusty, `docker exec` does not have `--workdir` option.